gsa/src/web/pages/portlists/component.js

```diff
@@ -216,7 +216,7 @@ class PortListComponent extends React.Component {
     }
 
     // reject port ranges with start value lower than end value
-    if (port_range_start > port_range_end) {
+    if (parseInt(port_range_start) > parseInt(port_range_end)) {
```

Comment:
Can you check if we can already parse this in the model? There are other comparisons in the portlist component that also seem to assume that port_range_start and port_range_end are integers.

Reply:
This concerns the _new_ inputs from the dialog that are compared and it's not linked to the model. `start` and `end` are Int in the model already.

Reply:
I see. Could you change the code in this handler method to just store both values as integers from the beginning. Some lines below they are parsed as int again.